pediatric_rootlets/crop_composed_images.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
```suggestion
"""
# The script requires the SCT conda environment to be activated (because we import the SCT's Image class):
 #    source ${SCT_DIR}/python/etc/profile.d/conda.sh
 #    conda activate venv_sct
"""

import os
```